client/src/components/lobby/LobbySeeks.tsx

```diff
@@ -1,15 +1,94 @@
+// tslint:disable:no-console
 import * as React from 'react';
+import styled from 'styled-components';
+import {CreatedGames} from '../../store/MockData';
+import { NavLink } from 'react-router-dom';
 
-class LobbySeeksComponent extends React.Component<any, any> {
+const Container = styled.div `
+grid-column-start: 2;
+grid-column-end: 3;
+grid-row-start: 2;
+grid-row-end: 3;
+background-color: white;
+opacity: 0.8;
+height: auto;
+width: auto;
+overflow-y: scroll;
+padding: 30px 40px 0px 40px;
+box-shadow: 5px 5px 30px black, -5px -3px 30px black;
+  
+`
+const ListItemMatch = styled.div `
+display: grid;
+grid-template-columns: 20% 20% 20% 20% 20%;
+border-bottom: 1px solid black;
+padding: 20px 15px;
+place-items: baseline;
+&:last-of-type{
+  border-bottom: none;
+}
+`
+const JoinBTN = styled.button `
+justify-self: end;
+padding: 15px;
+color: white;
+background-color: green; 
+border-radius: 10px;
+border: none;
+:focus {
+  outline-width: 0;
+}
+`
+interface IStateType  {
+  Player: string;
+}
+class LobbySeeksComponent extends React.Component<any, IStateType> {
+  constructor (props: any) {
+    super(props)
+    this.state = {       
+      
+      Player: "kalle",
+    };
+    this.handleClick.bind(this);
+}
+// function id from match routar oppoenent till rätt /await/gameID sida
+  public handleClick = (gameId: any) => {
```

Comment:
this function doesnt do anything at the moment, what will it be used for ? for routing because of the comment inside it?

Reply:
Removing this line (55) of  code crashes the page. I'll try to find a way of removing it securely.